DQM/TrackerRemapper/test/test_catch2_SiStripTkMaps.cc

```diff
@@ -13,10 +13,9 @@ TEST_CASE("SiStripTkMaps testing", "[SiStripTkMaps]") {
     gStyle->SetOptStat(0);
     SiStripTkMaps theMap("COLZA L");
     theMap.bookMap("testing SiStripTkMaps", "counts");
-    SiStripDetInfoFileReader reader_ = SiStripDetInfoFileReader(edm::FileInPath(k_geo).fullPath());
-    const std::map<uint32_t, SiStripDetInfoFileReader::DetInfo>& DetInfos = reader_.getAllData();
+    const auto detInfo = SiStripDetInfoFileReader::read(edm::FileInPath(k_geo).fullPath());
```

Comment:
can't it be `edm::FileInPath(SiStripDetInfoFileReader::kDefaultFile)` also here?

Reply:
Yes, thanks, I didn't notice the definition above

Reply:
thanks!